### PR TITLE
Changes to issue template for parser

### PR DESCRIPTION
**Pull request by progmatic-99**
- Issue template changed to make parser for meetup-manager work

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/propose-talk-workshop.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ assignees: ''
...
-2. Embedded Python and IOT - MicroPython, Python on Hardware, Robotics, Arduino and Raspberry Pi
-3. Culture and society - Diversity, health, productivity, workspace issues, privacy, community building, coding for causes
-4. Core Python - Language Features, Python Implementations, Extending Python and Standard Library, language internals
-5. Data Science, Machine Learning, and AI
-6. Desktop Applications - Qt, GTK+, Tkinter, Gnome, KDE, Accessibility
-7. Scientific Computing - Python usage in scientific computing and research. GIS, Mathematics, Simulations
-8. Developer tools and automation - Testing, CI/CD, Containers, Orchestration, Logging and Monitoring, Cloud, SRE
-9. Web development - Web, Apis, Microservices
-10. Networking and Security - Network Programming, Network Security and Encryption
+1. Game Design and 3D Modelling - Python in developing games, 3-D modelling and animation-
```
Comment (inovizz) [inline .github/ISSUE_TEMPLATE/propose-talk-workshop.md:14]: @progmatic-99 - these hyphens, in the end, won't make sense. Can you instead add a period(.) If that can help you write the parser.

File: .github/ISSUE_TEMPLATE/propose-talk-workshop.md [hunk 2]
```diff
@@ -39,12 +39,12 @@ One can submit a talk on anything related to Python or Open Source and below are
...
-1. Speaker Bio (Brief)
-2. Company/College
-3. Email
-4. Years of Exp
- -->
+<!-- Please do include the following things-
+1. Speaker Bio (Brief)-
+2. Company/College-
+3. Email-
+4. Years of Exp -->
```
Comment (inovizz) [inline .github/ISSUE_TEMPLATE/propose-talk-workshop.md:46]: let's move `-->` this to the next line and have a hyphen here `-`

**Comment by inovizz** [comment]
@GoelJatin  - Please review this as well.

File: .github/ISSUE_TEMPLATE/propose-talk-workshop.md (hunk 1 above)
Comment (progmatic-99) [reply .github/ISSUE_TEMPLATE/propose-talk-workshop.md:14]: I will test these changes out and share the results with you.

File: .github/ISSUE_TEMPLATE/propose-talk-workshop.md (hunk 2 above)
Comment (progmatic-99) [reply .github/ISSUE_TEMPLATE/propose-talk-workshop.md:46]: Moving --> to the next line introduces too much complexity to parse out the details we want.

File: .github/ISSUE_TEMPLATE/propose-talk-workshop.md (hunk 1 above)
Comment (progmatic-99) [reply .github/ISSUE_TEMPLATE/propose-talk-workshop.md:14]: I can change those hyphens to period(.)

Comment (GoelJatin) [reply .github/ISSUE_TEMPLATE/propose-talk-workshop.md:14]: Yeah actually hyphens are used in MD for Bulleted lists so better to check if we can do without it

**Review by GoelJatin — COMMENTED** [review]
Before we merge this, I see that we tried to write our own parser, did we also try with any of the existing options?

What are the challenges / issues we faced with those?

**Pull request closed without merging**